spec/spec_helper.rb

```diff
@@ -21,10 +21,15 @@
 require 'capybara/cuprite'
 
 Capybara.register_driver(:cuprite) do |app|
+  browser_options = {}.tap do |opts|
+    opts['no-sandbox'] = nil if ENV['CI']
+  end
```

Comment:
I had to do this to get around the timeout issues. Let me know if this is not something that we want to do.
It was recommended [here](https://stackoverflow.com/questions/57976909/how-to-fix-chrome-process-did-not-produce-websocket-url-within-2-seconds-when) and in Cuprit's readme for when a docker image is used. https://github.com/rubycdp/cuprite?tab=readme-ov-file#install